pkg/reconciler/autoscaling/kpa/scaler.go

```diff
@@ -61,7 +61,7 @@ const (
 	// race the Revision reconciler and scale down the pods before it can actually surface the pod errors.
 	// We should instead do pod failure diagnostics here immediately before scaling down the Deployment.
 	activationTimeoutBuffer = 10 * time.Second
-	activationTimeout       = time.Duration(rresources.ProgressDeadlineSeconds)*time.Second + activationTimeoutBuffer
+	activationTimeout       = deployment.ProgressDeadlineDefault + activationTimeoutBuffer
```

Comment:
should we not pick the actually configured value here?

Reply:
As described in the description, I want to split this in 2 parts